GVFS/GVFS.Installer.Mac/scripts/postinstall

```diff
@@ -1,4 +1,15 @@
 #!/bin/bash
+
+KEXTBUNDLEID="org.vfsforgit.PrjFSKext"
+KEXTPATH="/Library/Extensions/PrjFSKext.kext"
+isKextLoadedCmd="/usr/sbin/kextstat -l -b $KEXTBUNDLEID | wc -l"
+isKextLoaded=$(eval $isKextLoadedCmd)
```

Comment:
Might be good to include a comment about when we might expect the kext to be loaded (or not).

Reply:
Added a short description in the beginning.